docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -433,6 +433,10 @@ dependencyManagement {
 ----
 ====
 
+=== DefaultOpenForTrafficCount and its effect on EurekaServer warmup time
```

Comment:
Please use correct capitalisation in `defaultOpenForTrafficCount` and the \`\` signs around it.